pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSchemas.java

```diff
@@ -78,10 +78,14 @@ void run() throws Exception {
         @Parameter(description = "persistent://tenant/namespace/topic", required = true)
         private java.util.List<String> params;
 
+        @Parameter(names = { "-f",
+                "--force" }, description = "Delete schema forcefully")
```

Comment:
Same as the above comment, it's better to provide more information to understand what is the difference after use `--force`